package.json

```diff
@@ -25,7 +25,7 @@
     "cssnano": "^6.0.1",
     "dayjs": "^1.10.7",
     "dotenv": "2.0.0",
-    "markdown-to-jsx": "7.1.8",
+    "markdown-to-jsx": "7.3.2",
```

Comment:
Updated the library because the older version had a bug related to the `namedCodesToUnicode` option.